rosbag2_compression/test/rosbag2_compression/test_compressor.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gmock/gmock.h>
```

Comment:
IIUC https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

gmock/gmock.h needs to be after <string> as gmock.h is part of category 7: "Other libraries' .h files".

Reply:
Gmock is a c library, otherwise, cpplint complains about the order. (C, C++, Internal/other)

Reply:
Not that's not what the coding style states, the linter is confused because you're including headers with brackets and not quotes. Sorry, I should have made that clear in the 1st comment.

Reply:
Ah I see what you mean, let me address it in the upcoming decompressor PR.